client/pairs.js

```diff
@@ -6,7 +6,7 @@ import css from './styles/style.styl';
 
 // import components
 import App from './components/App';
-import Game from './components/Game';
+import Game from './components/Game/Game';
```

Comment:
Should consider renaming `./components/Game/Game.jsx` or `./components/Game/index.jsx`
Then you can still use `import Game from './components/Game';`, as the module resolution will inherit from it's folder name

Reply:
I have made this change, but now come across a minor issue in that searching for component files in VS code now becomes slightly more difficult because everything is named `index`